fyde-access-proxy/ha.md

```diff
@@ -18,6 +18,6 @@ nav_order: 9
 
 - Please check the [Network]({{ site.baseurl }}{% link fyde-access-proxy/req-net.md %}) requirements for load balancer information and diagram
 
-- NOTE: Redis Replication is out of the scope for this document
+- Redis Replication is out of the scope for this document
```

Comment:
What about *of* this document ?

Reply:
You're right. This has been copied around without proper proofing. Changing.